cloud/azure/modules/app/locals.tf

```diff
@@ -32,23 +31,27 @@ locals {
 
     SECRET_KEY = data.azurerm_key_vault_secret.app_secret_key.value
 
-    ADFS_SECRET    = data.azurerm_key_vault_secret.adfs_secret.value
-    ADFS_CLIENT_ID = data.azurerm_key_vault_secret.adfs_client_id.value
+    ADFS_SECRET                  = data.azurerm_key_vault_secret.adfs_secret.value
+    ADFS_CLIENT_ID               = data.azurerm_key_vault_secret.adfs_client_id.value
+    ADFS_DISCOVERY_URI           = data.azurerm_key_vault_secret.adfs_discovery_uri.value
+    APPLICANT_OIDC_CLIENT_SECRET = data.azurerm_key_vault_secret.adfs_secret.value
+    APPLICANT_OIDC_DISCOVERY_URI = data.azurerm_key_vault_secret.adfs_discovery_uri.value
+    APPLICANT_OIDC_CLIENT_ID     = data.azurerm_key_vault_secret.adfs_client_id.value
 
     # The values below are all defaulted to null. If SAML authentication is used, the values can be pulled from the
     # saml_keystore module
     LOGIN_RADIUS_KEYSTORE_NAME    = (var.saml_keystore_filename != null ? "/saml/${var.saml_keystore_filename}" : "")
     LOGIN_RADIUS_KEYSTORE_PASS    = var.saml_keystore_password
     LOGIN_RADIUS_PRIVATE_KEY_PASS = var.saml_private_key_password
 
-    CIVIFORM_API_SECRET_SALT = data.azurerm_key_vault_secret.api_secret_salt_key.value
+    CIVIFORM_API_SECRET_SALT           = data.azurerm_key_vault_secret.api_secret_salt_key.value
+    DATABASE_APPLY_DESTRUCTIVE_CHANGES = true
```

Comment:
I think this can be removed entirely? Because this variable is inside of this expression 

`app_settings` = merge({
}), var.civiform_server_environmental_variables)`